pkg/plugins/layout.go

```diff
@@ -54,7 +54,7 @@ const (
 
 func Layout(l logger.Interface, s schema.Stage, fs vfs.FS, console Console) error {
 	if s.Layout.Device == nil {
-		l.Debug("did not find any device in layout")
+		l.Debug("Device empty, not running layout plugin")
```

Comment:
this makes more sense, the other message was very weird with no explanation of why it appeared

Reply:
```suggestion
		l.Debug("Device field empty, skipping layout plugin")
```